src/rollup/index.ts

```diff
@@ -258,7 +258,9 @@ export default function rollup(
 						chunks.filter(chunk => chunk.entryModule).map(chunk => chunk.entryModule.id)
 					);
 
-					return createAddons(graph, outputOptions)
+					return graph.pluginDriver
+						.hookSeq('renderStart')
```

Comment:
Not 100% sure about the hookSeq vs hookParallel part but my feeling is to go for parallel in the first implementation simply because
* it is potentially faster
* going from parallel to seq later is not a breaking change while the other way around definitely is

And then wait if someone comes up with an important use case for seq instead of parallel. Unless you already have one?

Reply:
Updated